plugins/shared/PluginManifest.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace PluginShared;
+
+/// <summary>
+/// This class represents the OpenAI plugin manifest.
```

Comment:
Would be helpful to link the OpenAI manifest definition here: https://platform.openai.com/docs/plugins/getting-started/plugin-manifest